feeds/ipq807x/ipq807x/base-files/lib/upgrade/platform.sh

```diff
@@ -84,11 +84,17 @@ emmc_do_upgrade_bootconfig() {
 	fi
```

Comment:
emmc_do_upgrade_bootconfig is now incorrect call it something wf660 specific please